spring-cloud-gcp-samples/spring-cloud-gcp-sql-mysql-r2dbc-sample/src/test/java/com/example/SqlR2dbcMySqlSampleApplicationIntegrationTests.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2021-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfSystemProperty;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Simple integration test to verify the SQL sample application with MySQL.
+ *
+ */
+@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT, classes = { SqlApplication.class }, properties = {
+		"spring.cloud.gcp.sql.databaseName=code_samples_test_r2dbc_db",
+		"spring.cloud.gcp.sql.instanceConnectionName=spring-cloud-gcp-ci:us-central1:testmysql",
```

Comment:
We have those defined in `spring-cloud-gcp-samples/pom.xml` . (`spring.cloud.gcp.sql.instance-connection-name`, `spring.cloud.gcp.sql.database-name`). So it should be possible to omit these when running maven with `-Pspring-cloud-gcp-ci-it`. Omitting these also allows a developer to run against a personal project.

Reply:
I was previously using different databases for r2dbc and jdbc when we discussed this earlier (which was probably unnecessary). Modified the mySQL test to use the properties from the pom.xml. Thank you for the suggestion.